stempy/mask.cpp

```diff
@@ -7,19 +7,26 @@ using namespace std;
 
 namespace stempy {
 
-uint16_t* createAnnularMask(int rows, int columns, int innerRadius, int outerRadius) {
+uint16_t* createAnnularMask(int rows, int columns, int innerRadius,
+                            int outerRadius, int centerX, int centerY)
+{
   auto numberOfElements = rows*columns;
   auto mask = new uint16_t[numberOfElements]();
-  auto xCenter = round(rows/2.0);
-  auto yCenter = round(rows/2.0);
+
+  if (centerX < 0)
+    centerX = round(rows / 2.0);
+
+  if (centerY < 0)
+    centerY = round(rows / 2.0);
```

Comment:
Should one of these be calculated via the columns instead of the rows?

Reply:
yes